gridstatus/caiso/caiso.py

```diff
@@ -2512,10 +2527,25 @@ def get_nomogram_branch_shadow_prices_hasp_hourly(
             columns={
                 "NOMOGRAM_ID": "Location",
                 "PRC": "Price",
+                "NOMOGRAM_ID_XML": "Nomogram ID XML",
+                "CONSTRAINT_CAUSE": "Constraint Cause",
+                "MARKET_RUN_ID": "Market Run ID",
+                "GROUP": "Group",
             },
         )
 
-        return df[["Interval Start", "Interval End", "Location", "Price"]]
+        return df[
+            [
+                "Interval Start",
+                "Interval End",
+                "Location",
+                "Nomogram ID XML",
+                "Market Run ID",
+                "Constraint Cause",
+                "Price",
+                "Group",
+            ]
+        ]
 
     def get_nomogram_branch_shadow_price_forecast_15_min(
```

Comment:
Should this method name be renamed to `get_nomogram_branch_shadow_prices_forecast_15_min` (pluralize prices) for consistency?

Reply:
Hmm, I see your point about not maintaining backwards compatibility...I guess this is fine.